packages/docs/src/pages/guides/typescript.mdx

```diff
@@ -10,56 +10,54 @@ While most APIs in Theme UI should _just work_ in TypeScript,
 there are a few advanced use cases which will differ slightly.
 This guide is intended to cover those use cases.
 
-## Exact theme type
+## Extending the Theme
 
-The [`Theme` type](/theme-spec) represents all possible themes.
+If you need additional properties in the theme object,
+you can add them with [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation).
 
-It might be what you need when you're writing a library of reusable components
-or an app where the theme object is provided by the user and kept in the database.
-However, it's not the most convenient way to think about the theme,
-when building a blog or a landing page.
```

Comment:
Why is this being deleted? I’ve never used this functionality but it seems like useful context.

Reply:
I merged the "Exact theme type" and "Additional properties in the theme" sections but I can bring parts of it back.

Reply:
Though I don't think this is accurate:

> It might be what you need when... the theme object is provided by the user and kept in the database.

How would that work?

Reply:
I have no idea tbh—I’ve never tried anything like that

Reply:
> How would that work?

I did that.
Every tenant has it's own Theme object. They can customize it in Admin panel. You can't use the type of user's exact theme, because you only know it at runtime, just like you were building a component library, so you use `get` and optional chaining.

The app is serverside rendered, so there's no FOUC and you fetch the theme on initial visit.

Reply:
As it turns out, I might have a similar (but not server-side rendered) use case in the future. But I'm still not sure what the "takeaway" should be here.

I think that I'll add a note about it in the section describing the `get` helper. Thoughts?

Reply:
I added a piece here: https://github.com/hasparus/theme-ui/pull/4/commits/0452bdac4a5e415865662b3ecf229b88a37d90b9#diff-8ec9599b94c082ff1719d0791a7d7c2ee03ce4da1f699b626352f3595ed9f6b6R73